crates/mipsy_web/Cargo.toml

```diff
@@ -12,6 +12,7 @@ crate-type = ["cdylib", "rlib"]
 yew = "0.19.3"
 yew-agent = "0.1.0"
 gloo-file = "0.2.0"
+gloo-utils = { version = "0.1.5", features = ["serde"]}
```

Comment:
Where/ why is this being used? 

I can see that https://github.com/insou22/mipsy/pull/251/files#diff-c6996f00d87c10aab85be0b286226529ab6ebc71d826ef80d1d13d4f78e5d6f7R8 imports it, but what for?

Reply:
`JsValue::from_serde` is no longer implemented by wasm_bindgen.

Reply:
ooof course, lets depreciate an api and make a minor version bump

Reply:
no such thing as a minor bump on 0.x.y semver --- minor is considered breaking by default until 1.x.y